### PR TITLE
readme.md

### LINKED ISSUE #20: README
Make proper README

### PR DESCRIPTION
**Pull request by yashgautam9274**
Fixes #20

adding code of conduct and contributing guidelines link in readme file and many new thing in readme .md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,97 @@
+## WELCOME TO ALL
```
Comment (sidhu1012) [inline README.md:1]: Welcome All

File: README.md [hunk 2]
```diff
@@ -1,20 +1,97 @@
...
 
-# DataScientist 
+## DataScientist 
 DataScientist is a Python library in making to ease the life of a Data Scientist.
 
+# This project is a part of NJACK Codes 2020 👣
+  OPEN SOURCE CONTEST - NWoc Code
+
+  ⭐ [Check out the website of NJACK Winter of Code 2020](https://njackwintercode.github.io)
+
```
Comment (sidhu1012) [inline README.md:20]: Remove it.

File: README.md [hunk 3]
```diff
@@ -1,20 +1,97 @@
...
+## Quick Start :
```
Comment (sidhu1012) [inline README.md:21]: Instead of forking , cloning add steps to install the library

File: README.md (hunk 1 above)
Comment (yashgautam9274) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (yashgautam9274) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (yashgautam9274) [reply README.md]: done

**Comment by yashgautam9274** [comment]
Now all the changes done

**Comment by yashgautam9274** [comment]
when u merge my pull request

**Comment by sidhu1012** [comment]
Tomorrow, little occupied today.

File: README.md [hunk 4]
```diff
@@ -1,20 +1,106 @@
...
+![Maintained](https://img.shields.io/maintenance/yes/2020)
+![first-timers-only](https://img.shields.io/badge/first--timers--only-friendly-tomato.svg?style=flat&logo=git)
+![GitHub top language](https://img.shields.io/github/languages/top/Dacience/DataScientist?color=yellow&logo=python)
+
 
-# DataScientist 
+## DataScientist 
 DataScientist is a Python library in making to ease the life of a Data Scientist.
 
+## How to install library
```
Comment (sidhu1012) [inline README.md:16]: Remove this Section

Comment (yashgautam9274) [reply README.md]: done

File: README.md [hunk 5]
```diff
@@ -1,20 +1,64 @@
...
+
+We want to facilitate a healthy and constructive community behavior by adopting and enforcing our code of conduct.
+Please adhere toward our [Code-of-Conduct](code-of-conduct.md).
+
+## 🏆 Contributing
+
+Please read  [CONTRIBUTING.md](CONTRIBUTING..md)  for information on how to contribute to DataScientist.
+
 ## License
 [MIT](https://choosealicense.com/licenses/mit/)
```
Comment (sidhu1012) [inline README.md:58]: Replace this link with aur license, else LGTM and can be merged after this fix.

Comment (yashgautam9274) [reply README.md]: Done

Comment (yashgautam9274) [reply README.md]: i add link of our license file in readme.md

**Pull request merged**